en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc

```diff
@@ -433,7 +433,7 @@ The *Account* page appears, displaying a linking request from Anypoint Platform.
 . In the *Access Token* section, click *Authenticate in Anypoint*. +
 The Anypoint Platform login page appears.
 
-. Log in to Anypoint Platform. Ensure that the user logging into Anypoint Platform has the right xref:2.x@api-manager::access-client-app-id-task[permissions to create a client app].
+. Log in to Anypoint Platform. Ensure that the user logging into Anypoint Platform has the right xref:2.x@api-manager::access-client-app-id-task.adoc[permissions to create a client app^].
```

Comment:
Is this the MS Composer version? If so, no need for the caret since it's the same site.